testinfra/modules/file.py

```diff
@@ -69,7 +71,7 @@ def linked_to(self):
         >>> File("/var/lock").linked_to
         '/run/lock'
         """
-        return self.check_output("readlink -f %s", self.path)
+        return os.path.realpath(self.path)
```

Comment:
We can't do this for non local backend (paramiko, ssh, salt, docker, ...) neither for local backend with sudo.